pkgs/development/python-modules/hikari/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder
+, pythonRelaxDepsHook
+, pytest-runner
+, aiohttp
+, attrs
+, multidict
+, colorlog
+, pynacl
+, git
+, pytest
+, pytest-cov
+, pytest-randomly
+, pytest-asyncio
+, mock
+}:
+let
+  git_hash = "9ade5395ebf72350a869f559f7253fa429cad552";
+in
+buildPythonPackage rec {
+  pname = "hikari";
+  version = "2.0.0.dev120";
+
+  src = fetchFromGitHub {
+    owner = "hikari-py";
+    repo = "hikari";
+    rev = version;
+    hash = "sha256-+muM0uaGuOUAUO4nJ9ntIjlHSL5qPJd6M8nC9PVRAng=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    git
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    attrs
+    multidict
+    colorlog
+  ];
+
+  pythonRelaxDeps = true;
+
+  passthru.optional-dependencies = {
+    server = [ pynacl ];
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-runner
+    pytest-asyncio
+    pytest-cov
+    pytest-randomly
+    mock
+  ];
+
+  pythonImportChecks = [ "hikari" ];
+
+  disabled = pythonOlder "3.7";
+
+  postPatch = ''
+    # hikari wants a Git commit hash.
+    export SHA=$(git rev-list -n 1 ${version})
```

Comment:
It seems this environment variable is no longer needed.

Reply:
It is needed, see the line immediately after this one. I used a variable for readability.

Reply:
Isn't that the nix variable `git_hash`?
I could not find the exported `SHA` either in this derivation or the source code.

Reply:
Fixed, my bad.